websites/docs-smoke-test/src/content/views/custom-anchor.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Custom Anchor
+---
+
+## Expected Usage
+
+[Anchor expected usage links to the next line](#any-html-id-compatible-string-expected-usage)
+
+<Anchor name="any-html-id-compatible-string-expected-usage"/>
+
+### In mardown document
+
+```md
+<Anchor name="any-html-id-compatible-string-expected-usage" />
+```
+
+### Generated DOM
+
+```html
+<a name="any-html-id-compatible-string-expected-usage"></a>
+```
+
+## With Other Props
+
+[Anchor with other props links to the next line](#any-html-id-compatible-string-with-other-props)
+
+<Anchor name="any-html-id-compatible-string-with-other-props" id="any-html-id-compatible-string-with-other-props"/>
```

Comment:
is it possible to disallow this or automatically reset / delete the ID?  The intended behavior of the tag is to explicitly _not_ allow overriding the ID generation, i.e. to only allow  named anchors that by HTML spec have lower precedence than IDs. 

I know that it can't generally be turned off but I'd like to avoid encouraging it especially on this tag.  So in any case it should not be on this page because this page is the de facto documentation of what you can do.